docker-compose.yml

```diff
@@ -1,11 +1,11 @@
 version: "3.4"
 services:
   bitcoin.dnp.dappnode.eth:
-    image: "bitcoin.dnp.dappnode.eth:0.1.6"
+    image: "bitcoin.dnp.dappnode.eth:0.2.0"
```

Comment:
revert so we can do version bump properly and apply semver in another PR, the core team will handle it.